egi_notebooks_hub/d4science.py

```diff
@@ -80,30 +84,31 @@ def get(self):
             self.log.error("Unable to get the user or context from gcube?")
             raise web.HTTPError(403)
 
-        # discover WPS
-        self.log.info("Discover wps")
+        # discover WPS if enabled
         wps_endpoint = ""
-        discovery_url = url_concat(D4SCIENCE_DM_REGISTRY_URL, {"gcube-token": token})
-        req = HTTPRequest(discovery_url, method="GET")
-        try:
-            self.log.info("fetch")
-            resp = yield http_client.fetch(req)
-        except HTTPError as e:
-            # whatever, get out
-            self.log.warning("Something happened with gcube service: %s", e)
-            raise web.HTTPError(403)
-        root = ElementTree.fromstring(resp.body.decode("utf8", "replace"))
-        self.log.info("root %s", root)
-        for child in root.findall("Resource/Profile/AccessPoint/" "Interface/Endpoint"):
-            entry_name = child.attrib["EntryName"]
-            self.log.info("entry_name %s", entry_name)
-            if entry_name != "GetCapabilities":
-                wps_endpoint = child.text
-                self.log.info("WPS endpoint: %s", wps_endpoint)
-                break
-
+        if D4SCIENCE_DISCOVER_WPS.lower() in ['true', '1']:
+              
+            self.log.info("Discover wps")
+            discovery_url = url_concat(D4SCIENCE_DM_REGISTRY_URL, {"gcube-token": token})
+            req = HTTPRequest(discovery_url, method="GET")
+            try:
+                self.log.info("fetch")
```

Comment:
These should be `.debug` more than `.info`. They were there to help me back in the day debugging the call, but maybe not really needed anymore 

```suggestion
```